Error: Cannot find module 'fs-extra'

Issue: 

```
[adrian@pondal software]$ git clone https://github.com/stav/wgrep.git  
Cloning into 'wgrep'...
remote: Enumerating objects: 1, done.
remote: Counting objects: 100% (1/1), done.
remote: Total 38 (delta 0), reused 0 (delta 0), pack-reused 37
Unpacking objects: 100% (38/38), done.
[adrian@pondal software]$ cd wgrep/
[adrian@pondal wgrep]$ npm install -g
/home/adrian/.npm-global/bin/wgrep -> /home/adrian/.npm-global/lib/node_modules/wgrep/index.js

> puppeteer@1.14.0 install /home/adrian/software/wgrep/node_modules/puppeteer
> node install.js

Downloading Chromium r641577 - 109.8 Mb [====================] 100% 0.0s 
Chromium downloaded to /home/adrian/software/wgrep/node_modules/puppeteer/.local-chromium/linux-641577
+ wgrep@0.1.0
added 49 packages from 23 contributors in 10.005s
[adrian@pondal wgrep]$ wgrep test https://example.com
internal/modules/cjs/loader.js:651
    throw err;
    ^

Error: Cannot find module 'fs-extra'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at Object.<anonymous> (/home/adrian/software/wgrep/wgrep.js:9:13)
    at Module._compile (internal/modules/cjs/loader.js:799:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:810:10)
    at Module.load (internal/modules/cjs/loader.js:666:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:606:12)
    at Function.Module._load (internal/modules/cjs/loader.js:598:3)
```

Adding `fs-extra` to the list of dependencies in `package.json` and reinstalling solved the issue.

--- a/package.json
+++ b/package.json
@@ -22,6 +22,7 @@
     "co-prompt": "^1.0.0",
     "commander": "^2.16.0",
     "findit": "^2.0.0",
+    "fs-extra": "^8.0.1",
     "puppeteer": "^1.6.2"
   }
 }

